src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Plane.PlaneBuilder planeBuilder = new Plane.PlaneBuilder("Boing");
```

Comment:
try creating PlaneBuilder using default empty constructor, then set some fields and return a Plane object in the end

Reply:
done

Reply:
the default empty constructor is already present in each class unless it is overriden